OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/ResetPasswordAlertDialogFragment.java

```diff
@@ -0,0 +1,190 @@
+        args.putString(EXTRA_LOGIN_EMAIL, email);
+        fragment.setArguments(args);
+
+        if (context instanceof OnActivityResultListener) {
+            fragment.setListener((OnActivityResultListener) context);
+        } else {
+            throw new ClassCastException("Caller should implement the OnActivityResultListener interface.");
+        }
+        return fragment;
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        AlertDialog alertDialog = (AlertDialog) super.onCreateDialog(savedInstanceState);
+
+        editText = new TextInputEditText(getContext());
+        textInputLayout = new TextInputLayout(getContext());
+        textInputLayout.addView(editText);
+        textInputLayout.setHint(getContext().getString(R.string.email));
+
+        int horizontalPadding = Math.round(getResources().getDimension(R.dimen.abc_dialog_padding_material));
+        int verticalPadding = Math.round(getResources().getDimension(R.dimen.abc_dialog_padding_top_material));
+        textInputLayout.setPadding(horizontalPadding, verticalPadding, horizontalPadding, verticalPadding);
+
+        alertDialog.setView(textInputLayout);
+        editText.setText(getArguments().getString(EXTRA_LOGIN_EMAIL));
+
+        return alertDialog;
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+
+        /*
+         * There are two OnClickListeners associated with an alert dialog button:
+         * DialogInterface.OnClickListener and View.OnClickListener.
+         *
+         * DialogInterface.OnClickListener defines what clicking the button
+         * does before the normal flow takes over (before the dialog is auto-dismissed)
+         * This is what is generally set in AlertDialog.Builder.
+         *
+         * View.OnClickListener is the listener for the complete behavior of the button.
+         * This has to be set by finding the button (after it has been created),
+         * and overriding it. If you want to override the normal button click flow,
+         * do so by overriding View.OnClickListener.
+         *
+         * We want to prevent the dialog from automatically closing on positive button click,
+         * and letting submit() control the logic so, we are overriding View.OnClickListener here.
+         */
+        ((AlertDialog) getDialog()).getButton(DialogInterface.BUTTON_POSITIVE).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                submit(editText.getText().toString().trim());
+            }
+        });
+    }
+
+    public void setListener(@NonNull OnActivityResultListener listener) {
+        this.listener = listener;
+    }
+
+    public void showError(@NonNull String error) {
+        textInputLayout.setError(error);
+        textInputLayout.setErrorEnabled(true);
+        textInputLayout.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_FOCUSED);
```

Comment:
Now that we have the input layout in a separate file we can move the error enabled to layout xml. But we still need to fire the accessibility event so that TalkBalk reads out the new error if an old error is already being displayed.